### ISSUE TITLE
jspm inspect --forks no longer working

### ISSUE THREAD

**Issue by peisenmann** [jspm/jspm-cli#1011]
After updating to the latest jspm beta, `jspm inspect --forks` appears to no longer work correctly.

Steps: 
```shell
npm install jspm@beta
jspm dl-loader
jspm --version
# 0.16.0-beta.6
jspm --help
# See that jspm inspect [--forks] is still available
jspm inspect --forks

warn Enter a full package name of the format registry:repo.
```

**Comment by guybedford** [comment]
Thanks, fixed in https://github.com/jspm/jspm-cli/commit/11ad880a90cb0f2cc2477bd8891684ebee7dd8e8.

**Comment by guybedford** [comment]
Released in jspm@0.16.0-beta.7.
